src/syscheckd/src/whodata/win_whodata.c

```diff
@@ -1262,9 +1274,22 @@ int set_policies() {
     snprintf(command, OS_SIZE_1024, WPOL_BACKUP_COMMAND, WPOL_BACKUP_FILE);
 
     // Get the current policies
-    int wm_exec_ret_code = wm_exec(command, NULL, &result_code, 5, NULL);
-    if (wm_exec_ret_code || result_code) {
-        retval = 2;
+    int wm_exec_ret_code, i;
+    int retries = 5;
+    int timeout = 5;
+    for (i = 0; i <= retries; i++) {
+        wm_exec_ret_code = wm_exec(command, NULL, &result_code, timeout+i, NULL);
+        if (wm_exec_ret_code || result_code) {
+            retval = 2;
+            merror(FIM_AUDITPOL_ATTEMPT_FAIL, i+1);
+        }
+        else {
```

Comment:
[Coding style guide (C)](https://github.com/wazuh/wazuh/wiki/Coding-style-guide-%28C%29):
```suggestion
        } else {
```

Reply:
Addressed with https://github.com/wazuh/wazuh/commit/27cd442d425134662030a6e1d968de53bc80870b